src/main/java/seedu/exercise/logic/commands/EditExerciseDescriptor.java

```diff
@@ -0,0 +1,199 @@
+package seedu.exercise.logic.commands;
```

Comment:
since this class is extracted out, maybe put it inside a better named package. something like a `util` package or `builder` package inside the `commands` package

Reply:
In fact, rename this class to `EditExerciseBuilder`. Till now i have no idea what is a `Descriptor`

Reply:
Noted and changed.